### PR TITLE
Updating Readme for setup fixes from tmikeschu

### PR DESCRIPTION
**Pull request by rylanb**
## Overview
We could be more clear on the ways to set this up

## Details
Hopefully, very straightforward setup with step-by-step instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,61 +58,110 @@ can see the routes with ```rake routes```;
 # Preparing a Development Environment
 
 ## Prerequisites
+---
```
Comment (ericfreese) [inline README.md]: These `<hr>`s seem a little out of place to me. What's the thought process behind them?

File: README.md [hunk 2]
```diff
@@ -58,61 +58,110 @@ can see the routes with ```rake routes```;
...
+5. **In a new terminal tab/window:**
+
+  ```
+  bundle exec rake db:create
+  ```
+
+6. **If you receive an encoding error, go to db console and:**
+
+  ```
+  SET CLIENT_ENCODING TO 'WIN1252';
```
Comment (ericfreese) [inline README.md]: Is this a windows-specific issue? Do we really need to mention this? I've yet to come across someone running this on windows, but wouldn't utf-8 work ok?

File: README.md [hunk 3]
```diff
@@ -58,61 +58,110 @@ can see the routes with ```rake routes```;
...
+    database: bfr_webapp_db
+    username: bfr_webapp_user
+    password: CHANGEME
+    host: localhost
+    pool: 5
+    timeout: 5000
+  ```
+  **Warning:** _The database defined as "test" will be erased and re-generated from your  development database when you run "rake". Do not set this db to the same as development or  production._
+
+5. **In a new terminal tab/window:**
```
Comment (ericfreese) [inline README.md]: Why does this need to be in a new tab/window? Maybe this step would be better titled something like "Create the database"?

Comment (rylanb) [inline README.md]: Removed

File: README.md [hunk 4]
```diff
@@ -127,11 +174,30 @@ bundle exec rake foodrobot:send_weekly_summary
...
 
-Region admins will be created for the new region with email addresses based on the region's name. For example, if the region name is `San Francisco`, the created region admins will have email addresses: `admin-san-francisco@example.com`, `admin-san-francisco-2@example.com`, etc. Their passwords will all be `password`.
+**Note:** _Region admins will be created for the new region with email addresses based on the region's name. For example, if the region name is `San Francisco`, the created region admins will have email addresses: `admin-san-francisco@example.com`, `admin-san-francisco-2@example.com`, etc. Their passwords will all be `password`._
 
-The `db:sample_region` rake task does not create any `Log` records, so you'll have to run the rake task to generate logs based on the newly created schedule chains: `rake foodrobot:generate_logs`.
+**Note:** _The `db:sample_region` rake task does not create any `Log` records, so you'll have to run the rake task to generate logs based on the newly created schedule chains: `bundle exec rake foodrobot:generate_logs`._
```
Comment (ericfreese) [inline README.md]: Looks like a couple hanging spaces at the end of lines 200 and 198

File: README.md [hunk 5]
```diff
@@ -127,11 +174,30 @@ bundle exec rake foodrobot:send_weekly_summary
 
 ## Generating Sample Data
 
-Running the `db:sample_region` rake task will create a new `Region` in your database and populate it with a bunch of random data (volunteers, donors, recipients, schedule chains, etc.). For more info on what exactly is created, see `lib/sample_data/region_data.rb`.
+If you start with an empty schema, you'll want to start by creating a Volunteer user with the admin bit set, and a first region e.g.:
```
Comment (ericfreese) [inline README.md]: Could we remove this bit? All this stuff should be set up by running `rake db:seed` right? Would folks realistically want to set this up manually like this without running the seeds?

File: README.md (hunk 2 above)
Comment (rylanb) [inline README.md]: This was in Mike's Readme, so I left it as helpful troubleshooting info

File: README.md (hunk 5 above)
Comment (rylanb) [inline README.md]: Updated

File: README.md (hunk 2 above)
Comment (ericfreese) [inline README.md]: Ok, works for me.

**Pull request merged**